scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -522,9 +473,7 @@ mod tests {
             ),
         ];
 
-        test_replication(&get_uri(), schema, operations)
-            .await
-            .unwrap();
+        test_replication(schema, operations).await.unwrap();
```

Comment:
Getting rid of the `node_uri` parameter sounds like a change that should be put in a separate commit.